apps/bc-wallet-showcase-creator/components/Tabs-component.tsx

```diff
@@ -1,18 +1,20 @@
 'use client'
 
 import { Link, usePathname } from '@/i18n/routing'
+import { useTenant } from '@/providers/tenant-provider'
 import { useTranslations } from 'next-intl'
 
 export default function TabsComponent({ slug }: { slug: string }) {
```

Comment:
Maybe could be easier to pass as a prop, once the tenant on the provider can be null on the first load
```suggestion
export default function TabsComponent({ slug, tenant }: { slug: string, tenant: string }) {
```